ci/prow/periodic_config.go

```diff
@@ -297,3 +298,15 @@ func generateGoCoveragePeriodic(title string, repoName string, _ yaml.MapSlice)
 		return
 	}
 }
+
+// generateClearAlertsPeriodicJob generates the monitoring clear alerts job config.
+func generateClearAlertsPeriodicJob() {
+	var data periodicJobTemplateData
+	data.Base = newbaseProwJobTemplateData("knative/test-infra")
+	data.Base.Image = clearalertsDockerImage
+	data.PeriodicJobName = "ci-knative-monitoring-clear-alerts"
```

Comment:
ci-knative-test-infra-clear-alerts? Should we use the repo-name?

Reply:
For the clear alerts job, I think one job that runs every 30 minutes is sufficient. If we use the repo name in the job name, I looks to me like it's expecting n repo jobs to be created. 

Also, the clear alerts job is repo agnostic since it clear alerts based on job name regex and error pattern. We may be able to deduce the repo name from the job name, but I don't think we need this extra complexity to support clearing alerts for one specific repo only.

Reply:
I think the repo name is for where the job should be associated with. This is related to test-infra and I think it should be included in the name. We should follow the current pattern of naming.

Reply:
Thanks for clarifying it. Updated the ci job name to include test-infra.